components/HandleSubscription.ts

```diff
@@ -0,0 +1,54 @@
+import { baseUrl } from "../env";
+import { Dispatch, SetStateAction } from "react";
+
+export const handleSubscription = async (
+  isFollowed: boolean,
+  setIsFollowed: Dispatch<SetStateAction<boolean>>,
+  following: string,
+  subscriptionId: string,
+  setError: Dispatch<SetStateAction<string>>,
+) => {
+  const handleErrorResponse = (status: number) => {
```

Comment:
Gesamtes file bitte in extra function ordner machen -> ist eine funktion und keine componente, da hier kein JSX/TSX returned wird

Reply:
ist erledigt